addons/tmf_loadouts/loadouts/US_USMC_2003.hpp

```diff
@@ -0,0 +1,544 @@
+// If you want to base a loadout on an existing one, this repository contains them all:
+// https://github.com/ARCOMM/arc_misc/tree/master/addons/tmf_loadouts
+
+/* Dependencies:
+
+		* CUP
+		* Community Factions Project
+		* USM
+		* HLC M21
+		* FIR AWS
+
+*/
+
+//		Created by: AChesheireCat
+
+class baseMan {// Weaponless baseclass
+	displayName = "Unarmed";
+	// All randomized.
+	uniform[] = {
+		LIST_2("CUP_U_B_BDUv2_dirty_DCU"),
+		LIST_2("CUP_U_B_BDUv2_roll_dirty_DCU"),
+		LIST_2("CUP_U_B_BDUv2_gloves_dirty_DCU"),
+		LIST_2("CUP_U_B_BDUv2_dirty_M81"),
+		LIST_2("CUP_U_B_BDUv2_gloves_M81"),
+		"CUP_U_B_USMC_FROG3_DMARPAT"
+	};
+	vest[] = {"CUP_V_B_Interceptor_Rifleman_M81"};
+	backpack[] = {};
+	headgear[] = {};
+	goggles[] = {};
+	hmd[] = {};
+	// Leave empty to remove all. "Default" > leave original item.
+	faces[] = {};
+	// Leave empty to not change faces.
+	insignias[] = {};
+	// Leave empty to not change insignias
+
+	// All randomized
+	primaryWeapon[] = {};
+	scope[] = {};
+	bipod[] = {};
+	attachment[] = {};
+	silencer[] = {};
+	// Leave empty to remove all. "Default" for primaryWeapon > leave original weapon.
+
+	// Only *Weapons[] arrays are randomized
+	secondaryWeapon[] = {};
+	secondaryAttachments[] = {};
+	sidearmWeapon[] = {};
+	sidearmAttachments[] = {};
+	// Leave empty to remove all. "Default" for secondaryWeapon or sidearmWeapon > leave original weapon.
+
+	// These are added to the uniform or vest
+	magazines[] = {};
+	items[] = {
+		 MEDICAL_R ,
+		"ACE_Flashlight_XL50"
+	};
+	// These are added directly into their respective slots
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemRadio",
+		"ItemWatch"
+	};
+
+	// These are put into the backpack
+	backpackItems[] = {};
+
+	// Unit traits, see https://community.bistudio.com/wiki/setUnitTrait
+	traits[] = {};
+
+	// This is executed after unit init is complete. argument: _this = _unit.
+	code = "";
+};
+
+class r : baseMan {
+	displayName = "Rifleman";
+	headgear[] = {
+		"CUP_H_PASGTv2_NVG_DCU",
+		"CUP_H_USMC_LWH_ESS_DES",
+		"CUP_H_USMC_LWH_NVGMOUNT_DES"
+	};
+	backPack[] = {"CUP_B_SLA_Medicbag"};
+	primaryWeapon[] = {
+		"CUP_arifle_M16A2",
+		"CUP_arifle_M16A4_Base",
+		"CUP_arifle_M16A4_Grip"
+	};
+	scope[] = {
+		LIST_4(""),
+		"cup_optic_trijiconrx01_black"
+	};
+	attachment[] = {
+		"cup_acc_anpeq_2_grey",
+		"cup_acc_anpeq_2_black_top"
+	};
+	magazines[] = {
+		LIST_7("CUP_30Rnd_556x45_Stanag"),
+		LIST_2("CUP_30Rnd_556x45_Stanag_Tracer_Red"),
+		LIST_2("HandGrenade"),
+		"SmokeShell"
+	};
+};
+
+class g : r {
+	displayName = "Grenadier";
+	vest[] = {"CUP_V_B_Interceptor_Grenadier_M81"};
+	primaryWeapon[] = {
+		"CUP_arifle_M16A4_GL",
+		"CUP_arifle_M16A2_GL"
+	};
+	attachment[] = {"cup_acc_anpeq_2_grey"};
+	magazines[] += {
+		LIST_11("CUP_1Rnd_HEDP_M203"),
+		LIST_2("CUP_1Rnd_Smoke_M203")
+	};
+};
+
+class car : r {
+	displayName = "Carabinier";
```

Comment:
M4?

Reply:
Surprisingly it took until 2006 for all frontline units to receive the M4 Carbine. Even then, the M16A2, A3, and A4 variants were still in larger numbers until around 2010 wherein the M4 became the standard firearm.

(Also, the NIArms M4 is like, a pound heavier than the M16)